rpm/tracker.spec

```diff
@@ -63,71 +57,49 @@ Development files for %{name}.
 %autosetup -p1 -n %{name}-%{version}/upstream
 
 %build
-%meson -Dman=false -Ddocs=false -Dfunctional_tests=false \
-       -Dnetwork_manager=disabled -Dstemmer=disabled \
+%meson -Dman=false -Ddocs=false \
+       -Dstemmer=disabled \
        -Dunicode_support=icu \
-       -Dbash_completion=no \
-       -Dsystemd_user_services=%{_userunitdir}
+       -Dbash_completion=false \
+       -Dsystemd_user_services_dir=%{_userunitdir}
+
 %meson_build
 
 %install
 %meson_install
 
-# oneshot run in install
-install -D -m 755 %{SOURCE1} %{buildroot}/%{_oneshotdir}/tracker-configs.sh
-
-%find_lang %{name}
+%find_lang tracker3
 
 %post
 /sbin/ldconfig
-glib-compile-schemas   /usr/share/glib-2.0/schemas/
-if [ "$1" -ge 1 ]; then
-systemctl-user daemon-reload || :
-systemctl-user stop tracker-store.service || :
-add-oneshot --new-users --all-users tracker-configs.sh || :
-fi
```

Comment:
Should the service be set up on install?
```
%post
/sbin/ldconfig
%systemd_user_post tracker-xdg-portal-3
```

And before uninstallation:
```
%preun
%systemd_user_preun tracker-xdg-portal-3
```

Reply:
The portal service would need to be taken into use and that's better done separately. To some extend that could also need some tracker enhancements upstream: the file availability due to device mount status now always needs the FileSystem graph access and thus we can't filter graph access there.

Reply:
Isn't that a Flatpak thing? Do we want or need that? Perhaps it shouldn't be bundled at all, though left like this would allow third-party Flatpak deployments to run it.

Reply:
Kind of. Originating from the flatpak world though don't think it had dependencies on that side and perhaps could still be taken into use. Some depending code seems to be checking whether running inside flatpak, e.g. grilo-plugins checks /.flatpak-info and then sets the miner d-bus service based on that file.

So at this point it could indeed be dropped from the package, but as it's only 36,7kB here I thought it could be better to just include it but not run it.

Reply:
Ok, makes sense.